ui/src/main/kotlin/edu/artic/ui/BaseFragment.kt

```diff
@@ -3,22 +3,49 @@ package edu.artic.ui
 import android.os.Bundle
 import android.support.annotation.LayoutRes
 import android.support.v4.app.Fragment
+import android.support.v7.widget.Toolbar
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import com.fuzz.rx.DisposeBag
+import dagger.android.support.AndroidSupportInjection
 
-abstract class BaseFragment : Fragment(){
+abstract class BaseFragment : Fragment() {
```

Comment:
We're not clearing out the dispose bag on destroy. lets do that too  Ithink

Reply:
```
 override fun onDestroyView() {
        super.onDestroyView()
        disposeBag.clear()
}
```

Reply:
Added.